doc/run.md

```diff
@@ -91,7 +91,40 @@ In order to use the full feature set of Mayastor, some or all of the following c
   ```bash
   modprobe nbd nvmet nvmet_rdma nvme_fabrics nvme_tcp nvme_rdma nvme_loop
   ```
-* An NVMe device. (Typically via PCI-E through an standard slot or [M.2][m-dot-2] port)
+* For Asymmetric Namespace Access (ANA) support (early preview), the following kernel build configuration enabled:
+  + `CONFIG_NVME_MULTIPATH`: enables support for multipath access to NVMe subsystems
+
+  This is usually already enabled in distributions kernels, at least for RHEL/CentOS 8.2, Ubuntu 20.04 LTS, and SUSE Linux Enterprise 15.2.
```

Comment:
To keep things short and match above formatting, maybe this line can be deleted, and the modprobe instructions provided in a consistent format to the above?

```bash
modprobe nvme-core multipath=1
```

etc

Reply:
The kernel module option covers the case where `nvme-core.ko` is already loaded during boot, such as systems which use a NVMe drive. The multipath option cannot be modified at run-time.